aws/cloudformation/standalone/gen_ai_curriculum/monitoring/alarms/alarmConfigurations.ts

```diff
@@ -0,0 +1,101 @@
+import {ComparisonOperator} from '@aws-sdk/client-cloudwatch';
+
+import modelDescriptions from '../../../../../../apps/static/aichat/modelDescriptions.json';
+import {SNS_TOPIC} from '../constants';
+
+import {
+  createJobExecutionMetricStat,
+  createOpenaiMetricStat,
+} from './alarmHelpers';
+
+const modelIds = modelDescriptions.map((model: {id: string}) => model.id);
+
+export const openaiSafetyHighFailureRateConfiguration = {
+  AlarmName: 'genai_openai_safety_high_failure_rate',
+  AlarmDescription: 'genai_openai_safety_high_failure_rate',
+  ActionsEnabled: true,
+  OKActions: [],
+  AlarmActions: [SNS_TOPIC],
+  InsufficientDataActions: [],
+  EvaluationPeriods: 5,
+  DatapointsToAlarm: 5,
+  Threshold: 10,
```

Comment:
We were talking about possibly creating two different alarms of each type with a low and high threshold for different urgency levels. Is that something we should set up now, or wait until we have the SNS topics created?

Reply:
Will include in the follow-up. Thanks!